app/models/service_provider_request.rb

```diff
@@ -19,4 +19,8 @@ def ial
   def ial=(val)
     self.loa = val
   end
+
+  def ==(other)
+    to_json == other.to_json
```

Comment:
Personally I would do like a `.to_h` comparison instead of `to_json`? But it's probably fine as-is?

Reply:
There is no `to_h` method on a model.  Do you mean `attributes`?

Reply:
Yeah, but it doesn't matter much, if this works, it's fine